benchmark_runner/grafana/perf/extract_dashboard.py

```diff
@@ -0,0 +1,10 @@
+import re
+import argparse
+
+parser = argparse.ArgumentParser(description="Extract text between <<EOT and EOT in input file.")
```

Comment:
Is this supposed to extract it even if not a line of its own (in the latter case) and at the end of the line (in the former)?

Reply:
The terrafom will be failed if no line

Reply:
I mean if EOT is somehow present inside a line as opposed to being part of a here document.

Reply:
That generates by Terraform it also contains a strange prefix, this script do the work